src/include/homestore/checkpoint/cp.hpp

```diff
@@ -85,6 +85,9 @@ struct CP {
     cp_id_t m_cp_id;
     std::array< std::unique_ptr< CPContext >, (size_t)cp_consumer_t::SENTINEL > m_contexts;
     folly::SharedPromise< bool > m_comp_promise;
+#ifdef _PRERELEASE
+    std::atomic<bool> m_abrupt_cp{false};
+#endif
```

Comment:
We probably no longer need this code, will remove it and merge it with other PR that simulates crash.